Azure/azure-cli
# [SQL] Allow short mw policy names for mi

**Description**

Added a delighter that allows users to pass short MW policy names and CLI will construct the whole resource for a user

**Testing Guide**

az sql mi create -g mygroup -n myinstance -l mylocation -i -u myusername -p mypassword --subnet /subscriptions/{SubID}/resourceGroups/{ResourceGroup}/providers/Microsoft.Network/virtualNetworks/{VNETName}/subnets/{SubnetName} -m SQL_{Region}_{MaintenanceConfigName}

az sql mi update -g mygroup  -n myinstance -m SQL_Default

az sql mi update -g mygroup  -n myinstance -m SQL_{Region}_{MaintenanceConfigName}

---

This checklist is used to make sure that common guidelines for a pull request are followed.
